modules/poll/sheet/edit.js

```diff
@@ -0,0 +1,171 @@
+var nest = require('depnest')
+var extend = require('xtend')
+var Pickr = require('flatpickr')
+var spacetime = require('spacetime')
+
+var {Value, resolve, h, computed, when, map, Struct, Array: MutantArray} = require('mutant')
+
+var Poll = require('scuttle-poll')
+
+exports.gives = nest('poll.sheet.edit')
+
+exports.needs = nest({
+  'sheet.display': 'first',
+  'keys.sync.id': 'first',
+  'sbot.async.publish': 'first',
+  'sbot.obs.connection': 'first',
+  'about.obs.latestValue': 'first',
+  'blob.html.input': 'first',
+  'blob.sync.url': 'first',
+  'intl.sync.i18n': 'first'
+})
+
+exports.create = function (api) {
+  const i18n = api.intl.sync.i18n
+  return nest('poll.sheet.edit', function (id) {
+    var scuttlePoll = Poll(api.sbot.obs.connection)
+    api.sheet.display(close => {
+      const poll = Struct({
+        title: undefined,
+        body: undefined,
+        choices: MutantArray([Value(), Value(), Value()]),
+        closesAt: undefined
+      })
+
+      const page = h('PollNew -chooseOne', [
+        h('h2', {
+          style: {
+            'font-weight': 'normal'
+          }
+        }, [i18n('Create new Choose-one poll')]),
+        h('PollEditor', [
+          h('input.title', { 'ev-input': ev => poll.title.set(ev.target.value), placeholder: i18n('Choose a title') }, poll.title),
+          h('textarea.description', { 'ev-input': ev => poll.body.set(ev.target.value), placeholder: i18n('Describe the poll if you want') }, poll.body),
+
+          h('div.field -choices', [
+            h('div.inputs', [
+              map(poll.choices, (choice, index) => {
+                return h('input', { 'ev-input': ev => choice.set(ev.target.value), placeholder: i18n('Choice') }, choice)
+              }),
+              h('button', { 'ev-click': () => poll.choices.push(Value()) }, i18n('+ Add more choices'))
+            ])
+          ]),
+          h('input.date', {
+            placeholder: i18n('Choose date and time'),
```

Comment:
user feedback: what is this date and time for? is it when the poll closes?

Reply:
yes